pytest.ini

```diff
@@ -16,6 +16,11 @@ filterwarnings =
 
     ignore: the imp module is deprecated
 
+    # aca_view
+    ignore:There is no current event loop:DeprecationWarning
```

Comment:
This one might be a true warning (one that depends on changes in aca_view)  so it might not be a good idea to silence it

Reply:
Right, though the problem with not silencing it is that the tests aren't going to pass.  So this PR documents what I needed to have passing tests.  If you want to handle this on the aca_view side that's a different option.